<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleAdd</class>
 <widget class="QDialog" name="RoleAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>331</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 255);</string>
  </property>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>108</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/LCD_CS351/LCD_CS351/logo/check.png);
}
</string>
   </property>
   <property name="text">
    <string>系统设置</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>108</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/LCD_CS351/LCD_CS351/logo/check.png);
}
</string>
   </property>
   <property name="text">
    <string>编辑程序</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>168</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/LCD_CS351/LCD_CS351/logo/check.png);
}
</string>
   </property>
   <property name="text">
    <string>查看历史</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>228</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/LCD_CS351/LCD_CS351/logo/check.png);
}
</string>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>168</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/LCD_CS351/LCD_CS351/logo/check.png);
}
</string>
   </property>
   <property name="text">
    <string>作业指导</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>41</y>
     <width>71</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>角色：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_roleName">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>41</y>
     <width>181</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   font: 14pt &quot;黑体&quot;;
   color: rgb(248, 248, 255);
   border-style:solid;
   background-color: rgb(255, 102, 102);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   font: 14pt &quot;黑体&quot;;
   color: rgb(248, 248, 255);
   border-style:solid;
   background-color: rgb(51, 153, 255);
}
QPushButton:pressed{
	Qt::ToolButtonTextUnderIcon
}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/cha.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_wrong">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>6</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(250, 0, 0);</string>
   </property>
   <property name="text">
    <string>请填写角色名！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
   font: 14pt &quot;黑体&quot;;
   color: rgb(248, 248, 255);
   border-style:solid;
   background-color: rgb(250, 206, 16);
}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
